pkg/types/kubevirt/platform.go

```diff
@@ -0,0 +1,30 @@
+package kubevirt
+
+// Platform stores all the global configuration that all
+// machinesets use.
+type Platform struct {
+	// The Namespace in the infra cluster, which the control plane (master vms)
+	// and the compute (worker vms) are installed in
+	Namespace string `json:"namespace"`
+
+	// The Storage Class used in the infra cluster
+	StorageClass string `json:"storageClass"`
+
+	// NetworkName is the target network of all the network interfaces of the nodes.
+	NetworkName string `json:"networkName"`
+
+	// APIVIP is an IP which will be served by bootstrap and then pivoted masters, using keepalived
```

Comment:
This description is more of an implementation detail. The important part is that it is the VIP used for the external kube API.

Reply:
Fixed